sonic-xcvrd/scripts/xcvrd

```diff
@@ -220,6 +327,7 @@ def post_port_sfp_dom_info_to_db(stop=threading.Event()):
 
         post_port_sfp_info_to_db(logical_port_name, int_tbl, stop)
         post_port_dom_info_to_db(logical_port_name, dom_tbl, stop)
+        post_port_dom_threshold_info_to_db(logical_port_name, dom_tbl, stop)
```

Comment:
not quite sure here to use dom_tbl is a good idae, maybe should use int_tbl since the threshold is static if my understanding is correct.  @jleveque any comments?

Reply:
@jleveque @sridhar-ravindran I have the concern to mix the constant threshold values and the variables. would you please share your view?

Reply:
Hi @keboliu,
Since threshold values are coming under the dom category, i added threshold data in dom table.

Reply:
Hi @keboliu  @jleveque 
Since the threshold part comes under the dom category, i updated the dom_tbl.

Reply:
@jleveque I would like to get your view on this.

Reply:
Hi @keboliu, @jleveque 

In Show command, 
a) For dumping SFP info eeprom contents we use int_tbl(TRANSCEIVER_INFO)
b) For dumping DOM info eeprom contents we use dom_tbl (TRANSCEIVER_DOM_SENSOR)

My(@sridhar-ravindran) approach is since the thresholds that we are displaying corresponds to DOM data, i save the threshold information in dom_tbl. So dom thresholds will be displaying along with other dom data (sfpshow eeprom --dom)
Note:As @keboliu  suggested, we will read the threshold value only once (since it is static)
We will not read threshold value periodically like dom_data.

@keboliu  suggests, since the dom threshold data is static, we can save the threshold in int_tbl(TRANSCEIVER_INFO).

a) Then we will be displaying the dom_threshold data along with SFP info  (sfpshow eeprom)
                     (or)
b) We have to read both int_tbl and dom_tbl to display dom_threshold along with other dom_data (sfpshow eeprom --dom)

We need your inputs which approach to use, save dom_threshold data in dom_tbl or int_tbl?

Reply:
I understand both points here. The thresholds are related to DOM data, so it makes more sense to store in dom_tbl. As long as there isn't an efficiency issue here (i.e., we don't needlessly update the threshold values along with the DOM data, then I'm fine with either approach. From @sridhar-ravindran's most recent reply,

> @keboliu suggests, since the dom threshold data is static, we can save the threshold in int_tbl(TRANSCEIVER_INFO).

> a) Then we will be displaying the dom_threshold data along with SFP info (sfpshow eeprom)
(or)
b) We have to read both int_tbl and dom_tbl to display dom_threshold along with other dom_data (sfpshow eeprom --dom)

I think it makes more sense to store the threshold values in dom_tbl. That way `sfpshow eeprom` doesn't display threshold data and also we won't have to read from both int_tbl and dom_tbl to display dom_threshold along with other dom_data.